test/energy_scan/mult_and_spectra_plotter.py

```diff
@@ -471,7 +480,9 @@ def plotting(data1, data2, config_file, smash_code_version, output_folder):
                                        2212 : 0.938,
                                        3122 : 1.116,
                                        3312 : 1.321,
-                                       3334 : 1.672 }
+                                       3334 : 1.672,
+                                       1000010020 : 1.8756,
```

Comment:
Here are the deuterons again...

Reply:
Yep, Sigma0 plots should definitely be suppressed. Good catch!